tests/mgbench/benchmark.py

```diff
@@ -196,22 +538,68 @@ def filter_benchmarks(generators, patterns):
 # Filter out the generators.
 benchmarks = filter_benchmarks(generators, args.benchmarks)
 # Run all specified benchmarks.
-for dataset, tests in benchmarks:
+for dataset, queries in benchmarks:
+
+    # Save running configuration
+    workload_type = None
+    if len(args.mixed_workload) == 0:
+        workload_type = "Isolated"
+    elif len(args.mixed_workload) == 5:
+        workload_type = "Realistic"
+    else:
+        workload_type = "Mixed"
```

Comment:
For the mixed_workload types, I think it would be better to use an enum with specific values, which then can be nicely named and checked in these ifs...

Reply:
I am using a Class for this since Enum is limited in construction logic. Regarding the naming, on the graph bench we are using names: 

- Isolated  - all pure queries 
- Mixed - all queries + some percentage of  most likely write 
- Realistic - groups of queries in different combinations write 30% read 30% update 10% etc. 

I am pointing this out, bcs we have single -mixed-workload flag for two similar but different workloads. We are calling different names on the graph bench platform.  Maybe we can separate this, but obviously not super important.